src/hardware/pounder/timestamp.rs

```diff
@@ -56,11 +57,16 @@ impl Timestamper {
         let mut input_capture = capture_channel
             .into_input_capture(timers::tim8::CaptureSource1::TRC);
 
+        let prescaler = match batch_size {
```

Comment:
Just fix it to `Div8` as required for oversampling. The batch size shouldn't factor into it. I think this is an error that didn't have any consequence so far.

Reply:
I thought the intent here was to have a single timestamp per batch coming from pounder?

Reply:
You are right!
This one should be the batch size.
The prescaler I had in mind was the `etps()` that's (correctly) set in `setup()` to `Div4`. (Both are on the same timer.)